### PR TITLE
Added_project_content

### PR DESCRIPTION
**Pull request by andersonlebon**
In this project I have completed the following tasks:

- [ ]  Add the GitHub workflows to track linter errors in the PR,
- [ ]  Add linter to check for all CSS and HTML errors locally,
- [ ]  Add the HTML file to display the message "HELLO MICROVERS",
- [ ]  Add the CSS file for styling the message,
- [ ]  Add a `.gitignore` file to Ignore Unnecessary files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,28 +16,36 @@
...
 
 ## Built With
+
 This project is build with:
+
 - HTML
 - CSS
 
-[Link Demo](https://andersonlebon.github.io/hello-microvers/)
+[Link Demo](https://andersonlebon.github.io/hello-microvers-w2/)
```
Comment (jubaan) [inline README.md]: - [ ] README's are the most important part of our Repositories, we should always ensure that the links are working and take to the correct working content. Let's correct the live preview link right now is taking to a 404 error page.

**Review by jubaan — CHANGES_REQUESTED** [review]
Hi @andersonlebon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (andersonlebon) [reply README.md]: Sorry, but I have checked the live demo is working properly. Sometimes is giving errors. the problem is GitHub pages. but right now the link is working.

Thanks

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

## Optional suggestions to improve your project 🥇 

Please, consider:
- [OPTIONAL] The live demo shows the text `Hello microvers`, please, add the `e` at the end.
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**